pkg/kubelet/volumemanager/volume_manager.go

```diff
@@ -87,6 +88,10 @@ const (
 	// operation is waiting it only blocks other operations on the same device,
 	// other devices are not affected.
 	waitForAttachTimeout time.Duration = 10 * time.Minute
+
+	// metricsLoopSleepPeriod is the amount of time the metrics loop waits
+	// between successive executions.
+	metricsLoopSleepPeriod time.Duration = 10 * time.Second
```

Comment:
@gnufied, what do you think would be a good default value here? In hindsight, this seems too fast.

Reply:
Done.